pages/cancel/success.tsx

```diff
@@ -67,10 +76,10 @@ export default function Type(props) {
   );
 }
 
-export async function getServerSideProps(context) {
+export async function getServerSideProps(context: GetServerSidePropsContext) {
   const user = await prisma.user.findFirst({
     where: {
-      username: context.query.user,
+      username: context.query.user as string,
```

Comment:
```suggestion
      username: asStringOrThrow(context.query.user),
```

---

**Unrelated** sidenote - would be nicer if we used `context.query.username` everywhere across the app instead of `user` - maps better to the db and sometimes in the code `user` is a `User`-obj